helm/frost-server/templates/mqtt-ingress.yaml

```diff
@@ -1,10 +1,14 @@
-{{- if .Values.frost.mqtt.ingress.enabled -}}
+{{- if and .Values.frost.mqtt.ingress.enabled -}}
```

Comment:
This `and` currently has only one parameter.
I think this needs to have a `(not .Values.frost.mqtt.ingress.useSameAsHttp)` ? Or, considering `useSameAsHttp` is used further on, maybe the `and` should not be here?